src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperResolver.cs

```diff
@@ -19,7 +19,7 @@ internal abstract class TagHelperResolver : IWorkspaceService
 {
     private readonly ITelemetryReporter? _telemetryReporter;
 
-    public TagHelperResolver(ITelemetryReporter? telemetryReporter = null)
+    public TagHelperResolver(ITelemetryReporter? telemetryReporter)
```

Comment:
I'm curious why do this but not make it non-nullable? My guess would be to force whoever's using it to think about why they aren't supplying an `ITelemetryReporter` instead of default to not giving it?

Reply:
Yup! I just forgot to remove. Thanks :)